src/components/Tabs/Tabs.tsx

```diff
@@ -65,66 +61,37 @@ export const Tabs = ({
   onChange,
   ...other
 }: Props) => {
-  const ref = useRef<number | undefined>();
+  const getUID = useUIDSeed();
+  const activeTabPanelId = useUID();
   const headerRef = useRef<HTMLDivElement>(null);
   const [activeIndexState, setActiveIndexState] = useState(activeIndex);
   const [scrollableLeft, setScrollableLeft] = useState<boolean>(false);
   const [scrollableRight, setScrollableRight] = useState<boolean>(false);
 
-  /** Get all children that are Tab components. Ignore any others. */
-  const getTabs = useCallback(() => {
+  /** Children that are actually tabs. Any others are ignored. */
+  const tabs = useMemo(() => {
```

Comment:
Hat tip to @jeremiah-clothier for the idea.

By using `useMemo`, we only iterate `children` looking for `Tab`s once per render, instead of like 7 times per render.